ui/app/src/components/CreateDashboardDialog/CreateDashboardDialog.tsx

```diff
@@ -68,21 +84,46 @@ export const CreateDashboardDialog = (props: CreateDashboardDialogProps) => {
     <Dialog open={open} onClose={handleClose} aria-labelledby="confirm-dialog">
       <Dialog.Header>Create Dashboard</Dialog.Header>
       <Dialog.Content>
-        <TextField
-          required
-          margin="dense"
-          id="name"
-          label="Name"
-          type="text"
-          fullWidth
-          onChange={handleChange}
-          value={name}
-          error={!!error}
-          helperText={error}
-        />
+        <Stack gap={1}>
+          {projectOptions && projectOptions.length > 1 && (
+            <FormControl size="small" fullWidth>
+              <InputLabel>Project name</InputLabel>
```

Comment:
```suggestion
              <InputLabel id="project-name-id">Project name</InputLabel>
              <Select labelId="project-name-id">
```

would be good to  add `id` and `labelId` here for accessibility reasons